tests/utils.py

```diff
@@ -73,6 +79,7 @@ def setupdb():
             database=POSTGRES_BASE_DB,
             user=POSTGRES_USER,
             host=POSTGRES_HOST,
+            port=POSTGRES_PORT,
```

Comment:
I'm assuming `port=None` is the default behaviour. CI seems to be making good progress anyway...

Reply:
Yeah, indeed, psycopg2 is nicely designed that way